common/host/README

```diff
@@ -101,6 +100,29 @@ make ARCH=x86_64 menuconfig
 # "Device Drivers->Sound Card support -> Advanced linux sound Architecture -> ALSA for SoC audio support -> AMD Audio Coprocessor Support"
 # "Device Drivers->Sound Card support -> Advanced Linux Sound Architecture -> ALSA for SoC audio support -> AMD Audio Coprocessor-v3.x Support"
 
+# Apply patches specific to storage issues
+# Fix for nvme issue seen with Samsung and Toshiba storage seen on suspend/resume
+git am ~/device-intel-cic/common/host/storage/*.patch
+
+# Apply patches needed to debug kernel panic, mutex and hang issues
+# Apply patch needed to warm reset on kernel panic
+git am ~/device-intel-cic/common/host/debug/*.patch
+
+### Kernel Config Update
+make ARCH=x86_64 menuconfig
+
+# Enable configs to debug kernel panic, mutex and hang issues
```

Comment:
would you please add note to ensure customer disable kernel debug feature in product?